frontend/src/components/object/ObjectVersion.vue

```diff
@@ -39,15 +40,18 @@ const { getUserSearch } = storeToRefs(userStore);
 const tableData: Ref<Array<VersionDataSource>> = ref([]);
 
 // Actions
+const router = useRouter();
 const toast = useToast();
 
-// eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
-const showInfo = async (id: string) => {
-  // emit('show-object-info', id);
-};
-
-function onDeletedSuccess() {
+async function onDeletedSuccess() {
   toast.success('File deleted');
+  await versionStore.fetchVersions({ objectId: props.objectId });
+  router.push({ name: RouteNames.DETAIL_OBJECTS,
```

Comment:
this is good for when youre deleting the version you are on (viewing with versionId in url) 
but if youre deleting any other version , it doesnt seem to remove it from table (without a hard refresh) and doesn't really need to navigate to the latest.

maybe this would work:
```js
// if currently on latest version, re-load route
if (props.versionId === versionStore.findLatestVersionIdByObjectId(props.objectId) {
  router.push(...
 ```js